devicetypes/smartthings/aeon-multisensor.src/aeon-multisensor.groovy

```diff
@@ -102,25 +109,39 @@ def installed(){
 def updated(){
```

Comment:
just want to point out an issue with our platform, sometimes the devices are added silently, i.e. the updated() method is not called from the mobile app.
If this is required to be executed on install then this will have to be triggered from the install method too :(

Reply:
I think as written it's only expected to be called on a preference change.

Reply:
This was an issue I saw in other DTHs. Are we retroactively adding that as "best practice" now? @marstorp

Reply:
if it's only expected to be called on preference change I'm good with this, although I wonder if the user is presented with the choice to change the presences at install if it's a silent install.
But I'm fine with this as is :)

Reply:
@marstorp Shouldn't matter either way. While configure() should get called by the platform (which is perfect for when the device is powered via the USB port), this device will also receive the configure() during the Z-Wave WakeUpNotification if the device is battery powered.